ERROR_NOT_FOUND = Exception("Object not found.")